python/tvm/relay/frontend/tflite.py

```diff
@@ -1329,16 +1329,20 @@ def convert_fully_connected(self, op):
         input_tensor_shape = input_tensor.tensor.ShapeAsNumpy()
         weight_tensor_shape = weight_tensor.tensor.ShapeAsNumpy()
 
-        # reshape input tensor from N H W C to N H*W*C
-        input_size_per_batch = 1
-        for s in range(1, len(input_tensor_shape)):
-            input_size_per_batch *= input_tensor_shape[s]
-        assert input_size_per_batch == weight_tensor_shape[1], \
-            "input size and weight size are mismatched"
-        target_shape = tuple((input_tensor_shape[0], input_size_per_batch))
```

Comment:
Can you explain why this assert is wrong or needs to be removed in your covering note ?

Reply:
@u99127 : This assert is wrong assumption, as it always assumes first dimension is the batch size, while it has utterly failed to link between the input and weights, which is highly dependent when you operate in Fully_Connected or Dense operator.
As the frontend is just a transformation of an successfully deployed model, so we can safely assume the valid relationship exists! 
However this point will be evident when i put some comment to describe batch size computation as per your last comment. Thanks!